docs/src/controls.md

```diff
@@ -2,11 +2,10 @@
 
 Let's create a `slider` object:
 ```jldoctest demo1
-julia> using Gtk.ShortNames, GtkObservables
```

Comment:
Doesn't have to be fixed in this PR, but we'd want a path towards resolving doc errors too.

Reply:
Agreed -- it looks like we need to do some pattern matching to get them to pass. I will submit a new PR once I figure out how to do that.

Reply:
Yeah, definitely don't worry about that for this PR, unless you know of something that will make that really hard. One of us can tackle that later.